content/docs/explanation/models/data-models-full-example/index.en.md

```diff
@@ -67,91 +67,115 @@ For all track sections in our infrastructure, the `geo` and `sch` attributes are
 
 For most track sections, their `length` is proportional to what can be seen in the diagram. To preserve readability, exceptions were made for *TA6*, *TA7*, *TD0* and *TD1* (which are 10km and 25km).
 
-#### Track Section Links
 
-At the moment we only created track sections, which are not connected to each other (geospacial data is not used to deduce which tracks connect).
 
-`TrackSectionLinks` are used to connect two track sections together, just like a weld joint would in real life. In an OSRD simulation, a train can go from one track section to another only if they are connected by a `TrackSectionLink` (or by a `Switch`).
+#### Node
 
-To connect more than two `TrackSections` together, use the [`Switches`](#switches).
+A `Node` represents a node in the infrastructure. In an OSRD simulation, a train can only move from one section of track to another if they are linked by a node.
+A node can take one of two forms :
 
-In our infrastructure, since we chose to have our track sections as long as possible, we do not actually need to use `TrackSectionLinks`.
-`TrackSectionLinks` are always optional: two track sections connected by a link behave just like a single track section.
+**1) Switches**
 
-#### Switches
+Switches can be seen as a collection of track section links, partitioned into groups. Each group represents a state of the switch. Moving from one group to another can take time, and at most one link can be ready for use at any one time.
 
-A `Switch` represents just what you would expect: railway switches.
+In the real world, switches are not unique, but rather instances of existing models.
 
-Switches can be thought of as a collections of track section links, partitioned into groups. Each group represents for a switch state. Switching group can take time, and at most one link can be ready to use at a time.
+**2) Track section links**
 
-In the real world, switches are not unique, but rather instances of existing models.
-Thus, links and groups are not part of the switch itself, but in a `SwitchType` object, which is shared by switches of the same model.
+For the moment, we have only created track sections, which are not interconnected (geospatial data is not used to deduce which tracks are connected).
+
+Links are used to connect two track sections together, just as a weld joint would in real life. In an OSRD simulation, a train can only move from one section of track to another if they are connected by this type of node, the `link` (or by another `NodeType`).
+
+Whether for switches or track section links, links and groups are not part of the switch itself, but of a `Node` object, which is shared by switches in the same model.
 
-##### Switch Types
 
-`SwitchTypes` have two mandatory attributes:
+##### Node Types
+
+`NodeTypes` have two mandatory attributes:
 
 * `ports`: A list of port names. A port is an endpoint which can be connected to a track section.
 * `groups`: A mapping from group names to lists of links between two ports.
```

Comment:
I should add a better explanation of those terms, maybe including the word `branch`